pages/api/auth/[...nextauth].ts

```diff
@@ -1,14 +1,26 @@
 import Adapter from '@lib/nextAuthAdapter';
 import NextAuth from 'next-auth';
 import EmailProvider from 'next-auth/providers/email';
+import BoxyHQSAMLProvider from 'next-auth/providers/boxyhq-saml';
 import { validateEmailWithACL } from '@lib/utils';
+import { jacksonOptions as env } from '@lib/env';
 import { sessionName } from '@lib/constants';
 
 export default NextAuth({
   theme: {
     colorScheme: 'light',
   },
   providers: [
+    BoxyHQSAMLProvider({
+      authorization: { params: { scope: '' } },
+      issuer: env.externalUrl,
+      clientId: `tenant=${process.env.NEXT_PUBLIC_ADMIN_PORTAL_TENANT}&product=${process.env.NEXT_PUBLIC_ADMIN_PORTAL_PRODUCT}`,
```

Comment:
Can we read the `tenant` and `product` from the `adminPortalSSODefaults` instead?